ui/app/controllers/jobs/run.js

```diff
@@ -0,0 +1,65 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import { task } from 'ember-concurrency';
+import messageFromAdapterError from 'nomad-ui/utils/message-from-adapter-error';
+import localStorageProperty from 'nomad-ui/utils/properties/local-storage';
+
+export default Controller.extend({
+  parseError: null,
+  planError: null,
+  runError: null,
```

Comment:
What's the reason for separating these out? Would you ever get a parse error that could then proceed to a plan?